dev/lib/product_taxonomy/cli.rb

```diff
@@ -4,16 +4,7 @@ module ProductTaxonomy
   class Cli < Thor
     desc "dist", "Generate the taxonomy distribution"
     def dist
-      seconds = Benchmark.realtime do
-        source_data = YAML.safe_load_file("../data/attributes.yml")
-        values_model_index = Value.load_from_source(source_data: YAML.safe_load_file("../data/values.yml"))
-
-        Attribute.load_from_source(
-          source_data:,
-          values: values_model_index.hashed_by(:friendly_id),
-        )
```

Comment:
Did we want to keep this part? Since it's effective the dist generation bits right? Or I guess not quite yet

Reply:
Not quite yet, and the final version will likely be a single call into `Taxonomy.load_from_source` ([like in the prototype](https://github.com/Shopify/product-taxonomy/blob/dg/simplify/lib/product_taxonomy/cli.rb#L8)) rather than loading the different models here.